10:46:14.933 main  o.springframework.boot.SpringApplication Application startup failed
org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'manageController': Injection of autowired dependencies failed; nested exception is org.springframework.beans.factory.BeanCreationException: Could not autowire field: private service.DownloadExcelService controller.ManageController.downloadExcelService; nested exception is org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type [service.DownloadExcelService] found for dependency: expected at least 1 bean which qualifies as autowire candidate for this dependency. Dependency annotations: {@org.springframework.beans.factory.annotation.Autowired(required=true)}
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor.postProcessPropertyValues(AutowiredAnnotationBeanPostProcessor.java:334) ~[spring-beans-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.populateBean(AbstractAutowireCapableBeanFactory.java:1210) ~[spring-beans-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:537) ~[spring-beans-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:476) ~[spring-beans-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.springframework.beans.factory.support.AbstractBeanFactory$1.getObject(AbstractBeanFactory.java:303) ~[spring-beans-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:230) ~[spring-beans-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:299) ~[spring-beans-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:194) ~[spring-beans-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.preInstantiateSingletons(DefaultListableBeanFactory.java:755) ~[spring-beans-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:757) ~[spring-context-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:480) ~[spring-context-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:686) ~[spring-boot-1.2.3.RELEASE.jar:1.2.3.RELEASE]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:320) ~[spring-boot-1.2.3.RELEASE.jar:1.2.3.RELEASE]
	at org.springframework.boot.test.SpringApplicationContextLoader.loadContext(SpringApplicationContextLoader.java:101) [spring-boot-1.2.3.RELEASE.jar:1.2.3.RELEASE]
	at org.springframework.test.context.DefaultCacheAwareContextLoaderDelegate.loadContextInternal(DefaultCacheAwareContextLoaderDelegate.java:68) [spring-test-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.springframework.test.context.DefaultCacheAwareContextLoaderDelegate.loadContext(DefaultCacheAwareContextLoaderDelegate.java:86) [spring-test-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.springframework.test.context.DefaultTestContext.getApplicationContext(DefaultTestContext.java:72) [spring-test-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.springframework.test.context.support.DependencyInjectionTestExecutionListener.injectDependencies(DependencyInjectionTestExecutionListener.java:117) [spring-test-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.springframework.test.context.support.DependencyInjectionTestExecutionListener.prepareTestInstance(DependencyInjectionTestExecutionListener.java:83) [spring-test-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.springframework.test.context.TestContextManager.prepareTestInstance(TestContextManager.java:212) [spring-test-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.createTest(SpringJUnit4ClassRunner.java:200) [spring-test-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner$1.runReflectiveCall(SpringJUnit4ClassRunner.java:259) [spring-test-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12) [junit-4.12.jar:4.12]
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.methodBlock(SpringJUnit4ClassRunner.java:261) [spring-test-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:219) [spring-test-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:83) [spring-test-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290) [junit-4.12.jar:4.12]
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71) [junit-4.12.jar:4.12]
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288) [junit-4.12.jar:4.12]
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58) [junit-4.12.jar:4.12]
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268) [junit-4.12.jar:4.12]
	at org.springframework.test.context.junit4.statements.RunBeforeTestClassCallbacks.evaluate(RunBeforeTestClassCallbacks.java:61) [spring-test-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.springframework.test.context.junit4.statements.RunAfterTestClassCallbacks.evaluate(RunAfterTestClassCallbacks.java:68) [spring-test-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363) [junit-4.12.jar:4.12]
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.run(SpringJUnit4ClassRunner.java:163) [spring-test-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86) [.cp/:na]
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38) [.cp/:na]
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:539) [.cp/:na]
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:761) [.cp/:na]
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:461) [.cp/:na]
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:207) [.cp/:na]
Caused by: org.springframework.beans.factory.BeanCreationException: Could not autowire field: private service.DownloadExcelService controller.ManageController.downloadExcelService; nested exception is org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type [service.DownloadExcelService] found for dependency: expected at least 1 bean which qualifies as autowire candidate for this dependency. Dependency annotations: {@org.springframework.beans.factory.annotation.Autowired(required=true)}
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredFieldElement.inject(AutowiredAnnotationBeanPostProcessor.java:561) ~[spring-beans-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.springframework.beans.factory.annotation.InjectionMetadata.inject(InjectionMetadata.java:88) ~[spring-beans-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor.postProcessPropertyValues(AutowiredAnnotationBeanPostProcessor.java:331) ~[spring-beans-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	... 40 common frames omitted
Caused by: org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type [service.DownloadExcelService] found for dependency: expected at least 1 bean which qualifies as autowire candidate for this dependency. Dependency annotations: {@org.springframework.beans.factory.annotation.Autowired(required=true)}
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.raiseNoSuchBeanDefinitionException(DefaultListableBeanFactory.java:1301) ~[spring-beans-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.doResolveDependency(DefaultListableBeanFactory.java:1047) ~[spring-beans-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.resolveDependency(DefaultListableBeanFactory.java:942) ~[spring-beans-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredFieldElement.inject(AutowiredAnnotationBeanPostProcessor.java:533) ~[spring-beans-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	... 42 common frames omitted
10:46:14.939 main  o.s.test.context.TestContextManager Caught exception while allowing TestExecutionListener [org.springframework.test.context.support.DependencyInjectionTestExecutionListener@6c779568] to prepare test instance [test.DownlodExcekTest@1a4930cf]
java.lang.IllegalStateException: Failed to load ApplicationContext
	at org.springframework.test.context.DefaultCacheAwareContextLoaderDelegate.loadContext(DefaultCacheAwareContextLoaderDelegate.java:94) ~[spring-test-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.springframework.test.context.DefaultTestContext.getApplicationContext(DefaultTestContext.java:72) ~[spring-test-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.springframework.test.context.support.DependencyInjectionTestExecutionListener.injectDependencies(DependencyInjectionTestExecutionListener.java:117) ~[spring-test-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.springframework.test.context.support.DependencyInjectionTestExecutionListener.prepareTestInstance(DependencyInjectionTestExecutionListener.java:83) ~[spring-test-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.springframework.test.context.TestContextManager.prepareTestInstance(TestContextManager.java:212) ~[spring-test-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.createTest(SpringJUnit4ClassRunner.java:200) [spring-test-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner$1.runReflectiveCall(SpringJUnit4ClassRunner.java:259) [spring-test-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12) [junit-4.12.jar:4.12]
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.methodBlock(SpringJUnit4ClassRunner.java:261) [spring-test-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:219) [spring-test-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:83) [spring-test-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290) [junit-4.12.jar:4.12]
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71) [junit-4.12.jar:4.12]
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288) [junit-4.12.jar:4.12]
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58) [junit-4.12.jar:4.12]
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268) [junit-4.12.jar:4.12]
	at org.springframework.test.context.junit4.statements.RunBeforeTestClassCallbacks.evaluate(RunBeforeTestClassCallbacks.java:61) [spring-test-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.springframework.test.context.junit4.statements.RunAfterTestClassCallbacks.evaluate(RunAfterTestClassCallbacks.java:68) [spring-test-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363) [junit-4.12.jar:4.12]
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.run(SpringJUnit4ClassRunner.java:163) [spring-test-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86) [.cp/:na]
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38) [.cp/:na]
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:539) [.cp/:na]
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:761) [.cp/:na]
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:461) [.cp/:na]
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:207) [.cp/:na]
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'manageController': Injection of autowired dependencies failed; nested exception is org.springframework.beans.factory.BeanCreationException: Could not autowire field: private service.DownloadExcelService controller.ManageController.downloadExcelService; nested exception is org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type [service.DownloadExcelService] found for dependency: expected at least 1 bean which qualifies as autowire candidate for this dependency. Dependency annotations: {@org.springframework.beans.factory.annotation.Autowired(required=true)}
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor.postProcessPropertyValues(AutowiredAnnotationBeanPostProcessor.java:334) ~[spring-beans-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.populateBean(AbstractAutowireCapableBeanFactory.java:1210) ~[spring-beans-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:537) ~[spring-beans-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:476) ~[spring-beans-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.springframework.beans.factory.support.AbstractBeanFactory$1.getObject(AbstractBeanFactory.java:303) ~[spring-beans-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:230) ~[spring-beans-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:299) ~[spring-beans-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:194) ~[spring-beans-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.preInstantiateSingletons(DefaultListableBeanFactory.java:755) ~[spring-beans-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:757) ~[spring-context-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:480) ~[spring-context-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:686) ~[spring-boot-1.2.3.RELEASE.jar:1.2.3.RELEASE]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:320) ~[spring-boot-1.2.3.RELEASE.jar:1.2.3.RELEASE]
	at org.springframework.boot.test.SpringApplicationContextLoader.loadContext(SpringApplicationContextLoader.java:101) ~[spring-boot-1.2.3.RELEASE.jar:1.2.3.RELEASE]
	at org.springframework.test.context.DefaultCacheAwareContextLoaderDelegate.loadContextInternal(DefaultCacheAwareContextLoaderDelegate.java:68) ~[spring-test-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.springframework.test.context.DefaultCacheAwareContextLoaderDelegate.loadContext(DefaultCacheAwareContextLoaderDelegate.java:86) ~[spring-test-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	... 25 common frames omitted
Caused by: org.springframework.beans.factory.BeanCreationException: Could not autowire field: private service.DownloadExcelService controller.ManageController.downloadExcelService; nested exception is org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type [service.DownloadExcelService] found for dependency: expected at least 1 bean which qualifies as autowire candidate for this dependency. Dependency annotations: {@org.springframework.beans.factory.annotation.Autowired(required=true)}
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredFieldElement.inject(AutowiredAnnotationBeanPostProcessor.java:561) ~[spring-beans-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.springframework.beans.factory.annotation.InjectionMetadata.inject(InjectionMetadata.java:88) ~[spring-beans-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor.postProcessPropertyValues(AutowiredAnnotationBeanPostProcessor.java:331) ~[spring-beans-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	... 40 common frames omitted
Caused by: org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type [service.DownloadExcelService] found for dependency: expected at least 1 bean which qualifies as autowire candidate for this dependency. Dependency annotations: {@org.springframework.beans.factory.annotation.Autowired(required=true)}
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.raiseNoSuchBeanDefinitionException(DefaultListableBeanFactory.java:1301) ~[spring-beans-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.doResolveDependency(DefaultListableBeanFactory.java:1047) ~[spring-beans-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.resolveDependency(DefaultListableBeanFactory.java:942) ~[spring-beans-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredFieldElement.inject(AutowiredAnnotationBeanPostProcessor.java:533) ~[spring-beans-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	... 42 common frames omitted
10:47:15.044 main  o.s.test.context.TestContextManager Caught exception while allowing TestExecutionListener [org.springframework.test.context.support.DependencyInjectionTestExecutionListener@1329eff] to prepare test instance [test.DownlodExcekTest@72b1df49]
org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'test.DownlodExcekTest': Injection of autowired dependencies failed; nested exception is org.springframework.beans.factory.BeanCreationException: Could not autowire field: private org.springframework.web.context.WebApplicationContext test.DownlodExcekTest.context; nested exception is org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type [org.springframework.web.context.WebApplicationContext] found for dependency: expected at least 1 bean which qualifies as autowire candidate for this dependency. Dependency annotations: {@org.springframework.beans.factory.annotation.Autowired(required=true)}
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor.postProcessPropertyValues(AutowiredAnnotationBeanPostProcessor.java:334) ~[spring-beans-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.populateBean(AbstractAutowireCapableBeanFactory.java:1210) ~[spring-beans-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.autowireBeanProperties(AbstractAutowireCapableBeanFactory.java:385) ~[spring-beans-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.springframework.test.context.support.DependencyInjectionTestExecutionListener.injectDependencies(DependencyInjectionTestExecutionListener.java:118) ~[spring-test-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.springframework.test.context.support.DependencyInjectionTestExecutionListener.prepareTestInstance(DependencyInjectionTestExecutionListener.java:83) ~[spring-test-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.springframework.test.context.TestContextManager.prepareTestInstance(TestContextManager.java:212) ~[spring-test-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.createTest(SpringJUnit4ClassRunner.java:200) [spring-test-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner$1.runReflectiveCall(SpringJUnit4ClassRunner.java:259) [spring-test-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12) [junit-4.12.jar:4.12]
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.methodBlock(SpringJUnit4ClassRunner.java:261) [spring-test-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:219) [spring-test-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:83) [spring-test-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290) [junit-4.12.jar:4.12]
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71) [junit-4.12.jar:4.12]
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288) [junit-4.12.jar:4.12]
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58) [junit-4.12.jar:4.12]
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268) [junit-4.12.jar:4.12]
	at org.springframework.test.context.junit4.statements.RunBeforeTestClassCallbacks.evaluate(RunBeforeTestClassCallbacks.java:61) [spring-test-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.springframework.test.context.junit4.statements.RunAfterTestClassCallbacks.evaluate(RunAfterTestClassCallbacks.java:68) [spring-test-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363) [junit-4.12.jar:4.12]
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.run(SpringJUnit4ClassRunner.java:163) [spring-test-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86) [.cp/:na]
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38) [.cp/:na]
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:539) [.cp/:na]
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:761) [.cp/:na]
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:461) [.cp/:na]
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:207) [.cp/:na]
Caused by: org.springframework.beans.factory.BeanCreationException: Could not autowire field: private org.springframework.web.context.WebApplicationContext test.DownlodExcekTest.context; nested exception is org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type [org.springframework.web.context.WebApplicationContext] found for dependency: expected at least 1 bean which qualifies as autowire candidate for this dependency. Dependency annotations: {@org.springframework.beans.factory.annotation.Autowired(required=true)}
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredFieldElement.inject(AutowiredAnnotationBeanPostProcessor.java:561) ~[spring-beans-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.springframework.beans.factory.annotation.InjectionMetadata.inject(InjectionMetadata.java:88) ~[spring-beans-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor.postProcessPropertyValues(AutowiredAnnotationBeanPostProcessor.java:331) ~[spring-beans-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	... 26 common frames omitted
Caused by: org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type [org.springframework.web.context.WebApplicationContext] found for dependency: expected at least 1 bean which qualifies as autowire candidate for this dependency. Dependency annotations: {@org.springframework.beans.factory.annotation.Autowired(required=true)}
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.raiseNoSuchBeanDefinitionException(DefaultListableBeanFactory.java:1301) ~[spring-beans-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.doResolveDependency(DefaultListableBeanFactory.java:1047) ~[spring-beans-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.resolveDependency(DefaultListableBeanFactory.java:942) ~[spring-beans-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredFieldElement.inject(AutowiredAnnotationBeanPostProcessor.java:533) ~[spring-beans-4.1.6.RELEASE.jar:4.1.6.RELEASE]
	... 28 common frames omitted
14:10:47.377 http-nio-8093-exec-9  o.h.engine.jdbc.spi.SqlExceptionHelper Field 'id' doesn't have a default value
14:10:47.452 http-nio-8093-exec-9  config.GlobalDefaultExceptionHandler 异常的类型:  org.springframework.orm.jpa.JpaSystemException: could not execute statement; nested exception is org.hibernate.exception.GenericJDBCException: could not execute statement.异常出现类的名字:  org.springframework.orm.jpa.vendor.HibernateJpaDialect.异常方法的名字:  convertHibernateAccessException.异常出现的行数:  310
14:10:50.433 http-nio-8093-exec-9  o.h.engine.jdbc.spi.SqlExceptionHelper Field 'id' doesn't have a default value
14:10:50.436 http-nio-8093-exec-9  config.GlobalDefaultExceptionHandler 异常的类型:  org.springframework.orm.jpa.JpaSystemException: could not execute statement; nested exception is org.hibernate.exception.GenericJDBCException: could not execute statement.异常出现类的名字:  org.springframework.orm.jpa.vendor.HibernateJpaDialect.异常方法的名字:  convertHibernateAccessException.异常出现的行数:  310
14:10:54.193 http-nio-8093-exec-10  o.h.engine.jdbc.spi.SqlExceptionHelper Field 'id' doesn't have a default value
14:10:54.196 http-nio-8093-exec-10  config.GlobalDefaultExceptionHandler 异常的类型:  org.springframework.orm.jpa.JpaSystemException: could not execute statement; nested exception is org.hibernate.exception.GenericJDBCException: could not execute statement.异常出现类的名字:  org.springframework.orm.jpa.vendor.HibernateJpaDialect.异常方法的名字:  convertHibernateAccessException.异常出现的行数:  310
14:10:57.628 http-nio-8093-exec-2  o.h.engine.jdbc.spi.SqlExceptionHelper Field 'id' doesn't have a default value
14:10:57.632 http-nio-8093-exec-2  config.GlobalDefaultExceptionHandler 异常的类型:  org.springframework.orm.jpa.JpaSystemException: could not execute statement; nested exception is org.hibernate.exception.GenericJDBCException: could not execute statement.异常出现类的名字:  org.springframework.orm.jpa.vendor.HibernateJpaDialect.异常方法的名字:  convertHibernateAccessException.异常出现的行数:  310
14:11:42.087 http-nio-8093-exec-3  o.h.engine.jdbc.spi.SqlExceptionHelper Field 'id' doesn't have a default value
14:11:42.090 http-nio-8093-exec-3  config.GlobalDefaultExceptionHandler 异常的类型:  org.springframework.orm.jpa.JpaSystemException: could not execute statement; nested exception is org.hibernate.exception.GenericJDBCException: could not execute statement.异常出现类的名字:  org.springframework.orm.jpa.vendor.HibernateJpaDialect.异常方法的名字:  convertHibernateAccessException.异常出现的行数:  310
14:13:43.815 http-nio-8093-exec-10  config.GlobalDefaultExceptionHandler 异常的类型:  java.lang.NumberFormatException: For input string: "null".异常出现类的名字:  java.lang.NumberFormatException.异常方法的名字:  forInputString.异常出现的行数:  65
14:13:44.813 http-nio-8093-exec-8  config.GlobalDefaultExceptionHandler 异常的类型:  java.lang.NumberFormatException: For input string: "null".异常出现类的名字:  java.lang.NumberFormatException.异常方法的名字:  forInputString.异常出现的行数:  65
14:13:45.326 http-nio-8093-exec-9  config.GlobalDefaultExceptionHandler 异常的类型:  java.lang.NumberFormatException: For input string: "null".异常出现类的名字:  java.lang.NumberFormatException.异常方法的名字:  forInputString.异常出现的行数:  65
14:13:45.533 http-nio-8093-exec-6  config.GlobalDefaultExceptionHandler 异常的类型:  java.lang.NumberFormatException: For input string: "null".异常出现类的名字:  java.lang.NumberFormatException.异常方法的名字:  forInputString.异常出现的行数:  65
14:13:45.725 http-nio-8093-exec-2  config.GlobalDefaultExceptionHandler 异常的类型:  java.lang.NumberFormatException: For input string: "null".异常出现类的名字:  java.lang.NumberFormatException.异常方法的名字:  forInputString.异常出现的行数:  65
14:13:45.902 http-nio-8093-exec-10  config.GlobalDefaultExceptionHandler 异常的类型:  java.lang.NumberFormatException: For input string: "null".异常出现类的名字:  java.lang.NumberFormatException.异常方法的名字:  forInputString.异常出现的行数:  65
14:13:46.070 http-nio-8093-exec-8  config.GlobalDefaultExceptionHandler 异常的类型:  java.lang.NumberFormatException: For input string: "null".异常出现类的名字:  java.lang.NumberFormatException.异常方法的名字:  forInputString.异常出现的行数:  65
14:13:46.245 http-nio-8093-exec-5  config.GlobalDefaultExceptionHandler 异常的类型:  java.lang.NumberFormatException: For input string: "null".异常出现类的名字:  java.lang.NumberFormatException.异常方法的名字:  forInputString.异常出现的行数:  65
14:14:20.417 http-nio-8093-exec-9  o.h.engine.jdbc.spi.SqlExceptionHelper Field 'id' doesn't have a default value
14:14:20.422 http-nio-8093-exec-9  config.GlobalDefaultExceptionHandler 异常的类型:  org.springframework.orm.jpa.JpaSystemException: could not execute statement; nested exception is org.hibernate.exception.GenericJDBCException: could not execute statement.异常出现类的名字:  org.springframework.orm.jpa.vendor.HibernateJpaDialect.异常方法的名字:  convertHibernateAccessException.异常出现的行数:  310
14:14:21.231 http-nio-8093-exec-2  o.h.engine.jdbc.spi.SqlExceptionHelper Field 'id' doesn't have a default value
14:14:21.235 http-nio-8093-exec-2  config.GlobalDefaultExceptionHandler 异常的类型:  org.springframework.orm.jpa.JpaSystemException: could not execute statement; nested exception is org.hibernate.exception.GenericJDBCException: could not execute statement.异常出现类的名字:  org.springframework.orm.jpa.vendor.HibernateJpaDialect.异常方法的名字:  convertHibernateAccessException.异常出现的行数:  310
14:14:21.441 http-nio-8093-exec-1  o.h.engine.jdbc.spi.SqlExceptionHelper Field 'id' doesn't have a default value
14:14:21.446 http-nio-8093-exec-1  config.GlobalDefaultExceptionHandler 异常的类型:  org.springframework.orm.jpa.JpaSystemException: could not execute statement; nested exception is org.hibernate.exception.GenericJDBCException: could not execute statement.异常出现类的名字:  org.springframework.orm.jpa.vendor.HibernateJpaDialect.异常方法的名字:  convertHibernateAccessException.异常出现的行数:  310
14:14:21.615 http-nio-8093-exec-3  o.h.engine.jdbc.spi.SqlExceptionHelper Field 'id' doesn't have a default value
14:14:21.619 http-nio-8093-exec-3  config.GlobalDefaultExceptionHandler 异常的类型:  org.springframework.orm.jpa.JpaSystemException: could not execute statement; nested exception is org.hibernate.exception.GenericJDBCException: could not execute statement.异常出现类的名字:  org.springframework.orm.jpa.vendor.HibernateJpaDialect.异常方法的名字:  convertHibernateAccessException.异常出现的行数:  310
14:14:21.790 http-nio-8093-exec-7  o.h.engine.jdbc.spi.SqlExceptionHelper Field 'id' doesn't have a default value
14:14:21.794 http-nio-8093-exec-7  config.GlobalDefaultExceptionHandler 异常的类型:  org.springframework.orm.jpa.JpaSystemException: could not execute statement; nested exception is org.hibernate.exception.GenericJDBCException: could not execute statement.异常出现类的名字:  org.springframework.orm.jpa.vendor.HibernateJpaDialect.异常方法的名字:  convertHibernateAccessException.异常出现的行数:  310
14:14:21.967 http-nio-8093-exec-9  o.h.engine.jdbc.spi.SqlExceptionHelper Field 'id' doesn't have a default value
14:14:21.971 http-nio-8093-exec-9  config.GlobalDefaultExceptionHandler 异常的类型:  org.springframework.orm.jpa.JpaSystemException: could not execute statement; nested exception is org.hibernate.exception.GenericJDBCException: could not execute statement.异常出现类的名字:  org.springframework.orm.jpa.vendor.HibernateJpaDialect.异常方法的名字:  convertHibernateAccessException.异常出现的行数:  310
14:14:22.143 http-nio-8093-exec-2  o.h.engine.jdbc.spi.SqlExceptionHelper Field 'id' doesn't have a default value
14:14:22.146 http-nio-8093-exec-2  config.GlobalDefaultExceptionHandler 异常的类型:  org.springframework.orm.jpa.JpaSystemException: could not execute statement; nested exception is org.hibernate.exception.GenericJDBCException: could not execute statement.异常出现类的名字:  org.springframework.orm.jpa.vendor.HibernateJpaDialect.异常方法的名字:  convertHibernateAccessException.异常出现的行数:  310
14:16:56.020 http-nio-8093-exec-5  o.h.engine.jdbc.spi.SqlExceptionHelper Field 'id' doesn't have a default value
14:16:56.025 http-nio-8093-exec-5  config.GlobalDefaultExceptionHandler 异常的类型:  org.springframework.orm.jpa.JpaSystemException: could not execute statement; nested exception is org.hibernate.exception.GenericJDBCException: could not execute statement.异常出现类的名字:  org.springframework.orm.jpa.vendor.HibernateJpaDialect.异常方法的名字:  convertHibernateAccessException.异常出现的行数:  310
14:23:27.174 http-nio-8093-exec-8  o.h.engine.jdbc.spi.SqlExceptionHelper Field 'id' doesn't have a default value
14:23:27.185 http-nio-8093-exec-8  config.GlobalDefaultExceptionHandler 异常的类型:  org.springframework.orm.jpa.JpaSystemException: could not execute statement; nested exception is org.hibernate.exception.GenericJDBCException: could not execute statement.异常出现类的名字:  org.springframework.orm.jpa.vendor.HibernateJpaDialect.异常方法的名字:  convertHibernateAccessException.异常出现的行数:  310
14:23:29.015 http-nio-8093-exec-10  o.h.engine.jdbc.spi.SqlExceptionHelper Field 'id' doesn't have a default value
14:23:29.019 http-nio-8093-exec-10  config.GlobalDefaultExceptionHandler 异常的类型:  org.springframework.orm.jpa.JpaSystemException: could not execute statement; nested exception is org.hibernate.exception.GenericJDBCException: could not execute statement.异常出现类的名字:  org.springframework.orm.jpa.vendor.HibernateJpaDialect.异常方法的名字:  convertHibernateAccessException.异常出现的行数:  310
14:23:29.230 http-nio-8093-exec-2  o.h.engine.jdbc.spi.SqlExceptionHelper Field 'id' doesn't have a default value
14:23:29.235 http-nio-8093-exec-2  config.GlobalDefaultExceptionHandler 异常的类型:  org.springframework.orm.jpa.JpaSystemException: could not execute statement; nested exception is org.hibernate.exception.GenericJDBCException: could not execute statement.异常出现类的名字:  org.springframework.orm.jpa.vendor.HibernateJpaDialect.异常方法的名字:  convertHibernateAccessException.异常出现的行数:  310
14:23:29.413 http-nio-8093-exec-4  o.h.engine.jdbc.spi.SqlExceptionHelper Field 'id' doesn't have a default value
14:23:29.418 http-nio-8093-exec-4  config.GlobalDefaultExceptionHandler 异常的类型:  org.springframework.orm.jpa.JpaSystemException: could not execute statement; nested exception is org.hibernate.exception.GenericJDBCException: could not execute statement.异常出现类的名字:  org.springframework.orm.jpa.vendor.HibernateJpaDialect.异常方法的名字:  convertHibernateAccessException.异常出现的行数:  310
14:23:29.588 http-nio-8093-exec-7  o.h.engine.jdbc.spi.SqlExceptionHelper Field 'id' doesn't have a default value
14:23:29.593 http-nio-8093-exec-7  config.GlobalDefaultExceptionHandler 异常的类型:  org.springframework.orm.jpa.JpaSystemException: could not execute statement; nested exception is org.hibernate.exception.GenericJDBCException: could not execute statement.异常出现类的名字:  org.springframework.orm.jpa.vendor.HibernateJpaDialect.异常方法的名字:  convertHibernateAccessException.异常出现的行数:  310
14:23:29.756 http-nio-8093-exec-8  o.h.engine.jdbc.spi.SqlExceptionHelper Field 'id' doesn't have a default value
14:23:29.760 http-nio-8093-exec-8  config.GlobalDefaultExceptionHandler 异常的类型:  org.springframework.orm.jpa.JpaSystemException: could not execute statement; nested exception is org.hibernate.exception.GenericJDBCException: could not execute statement.异常出现类的名字:  org.springframework.orm.jpa.vendor.HibernateJpaDialect.异常方法的名字:  convertHibernateAccessException.异常出现的行数:  310
14:25:44.967 http-nio-8093-exec-4  o.h.engine.jdbc.spi.SqlExceptionHelper Field 'id' doesn't have a default value
14:25:44.974 http-nio-8093-exec-4  config.GlobalDefaultExceptionHandler 异常的类型:  org.springframework.orm.jpa.JpaSystemException: could not execute statement; nested exception is org.hibernate.exception.GenericJDBCException: could not execute statement.异常出现类的名字:  org.springframework.orm.jpa.vendor.HibernateJpaDialect.异常方法的名字:  convertHibernateAccessException.异常出现的行数:  310
14:26:16.670 http-nio-8093-exec-10  config.GlobalDefaultExceptionHandler 异常的类型:  org.springframework.orm.jpa.JpaSystemException: The database returned no natively generated identity value; nested exception is org.hibernate.HibernateException: The database returned no natively generated identity value.异常出现类的名字:  org.springframework.orm.jpa.vendor.HibernateJpaDialect.异常方法的名字:  convertHibernateAccessException.异常出现的行数:  310
14:28:08.014 http-nio-8093-exec-1  config.GlobalDefaultExceptionHandler 异常的类型:  org.springframework.orm.jpa.JpaSystemException: The database returned no natively generated identity value; nested exception is org.hibernate.HibernateException: The database returned no natively generated identity value.异常出现类的名字:  org.springframework.orm.jpa.vendor.HibernateJpaDialect.异常方法的名字:  convertHibernateAccessException.异常出现的行数:  310
14:28:09.076 http-nio-8093-exec-3  config.GlobalDefaultExceptionHandler 异常的类型:  org.springframework.orm.jpa.JpaSystemException: The database returned no natively generated identity value; nested exception is org.hibernate.HibernateException: The database returned no natively generated identity value.异常出现类的名字:  org.springframework.orm.jpa.vendor.HibernateJpaDialect.异常方法的名字:  convertHibernateAccessException.异常出现的行数:  310
14:28:09.555 http-nio-8093-exec-5  config.GlobalDefaultExceptionHandler 异常的类型:  org.springframework.orm.jpa.JpaSystemException: The database returned no natively generated identity value; nested exception is org.hibernate.HibernateException: The database returned no natively generated identity value.异常出现类的名字:  org.springframework.orm.jpa.vendor.HibernateJpaDialect.异常方法的名字:  convertHibernateAccessException.异常出现的行数:  310
14:28:09.716 http-nio-8093-exec-7  config.GlobalDefaultExceptionHandler 异常的类型:  org.springframework.orm.jpa.JpaSystemException: The database returned no natively generated identity value; nested exception is org.hibernate.HibernateException: The database returned no natively generated identity value.异常出现类的名字:  org.springframework.orm.jpa.vendor.HibernateJpaDialect.异常方法的名字:  convertHibernateAccessException.异常出现的行数:  310
14:28:09.906 http-nio-8093-exec-8  config.GlobalDefaultExceptionHandler 异常的类型:  org.springframework.orm.jpa.JpaSystemException: The database returned no natively generated identity value; nested exception is org.hibernate.HibernateException: The database returned no natively generated identity value.异常出现类的名字:  org.springframework.orm.jpa.vendor.HibernateJpaDialect.异常方法的名字:  convertHibernateAccessException.异常出现的行数:  310
14:33:00.456 http-nio-8093-exec-8  o.h.engine.jdbc.spi.SqlExceptionHelper Field 'id' doesn't have a default value
14:33:00.468 http-nio-8093-exec-8  config.GlobalDefaultExceptionHandler 异常的类型:  org.springframework.orm.jpa.JpaSystemException: could not execute statement; nested exception is org.hibernate.exception.GenericJDBCException: could not execute statement.
14:33:00.468 http-nio-8093-exec-8  config.GlobalDefaultExceptionHandler 异常出现类的名字:  org.springframework.orm.jpa.vendor.HibernateJpaDialect.
14:33:00.468 http-nio-8093-exec-8  config.GlobalDefaultExceptionHandler 异常方法的名字:  convertHibernateAccessException.
14:33:00.468 http-nio-8093-exec-8  config.GlobalDefaultExceptionHandler 异常出现的行数:  310
14:33:01.400 http-nio-8093-exec-10  o.h.engine.jdbc.spi.SqlExceptionHelper Field 'id' doesn't have a default value
14:33:01.405 http-nio-8093-exec-10  config.GlobalDefaultExceptionHandler 异常的类型:  org.springframework.orm.jpa.JpaSystemException: could not execute statement; nested exception is org.hibernate.exception.GenericJDBCException: could not execute statement.
14:33:01.405 http-nio-8093-exec-10  config.GlobalDefaultExceptionHandler 异常出现类的名字:  org.springframework.orm.jpa.vendor.HibernateJpaDialect.
14:33:01.405 http-nio-8093-exec-10  config.GlobalDefaultExceptionHandler 异常方法的名字:  convertHibernateAccessException.
14:33:01.406 http-nio-8093-exec-10  config.GlobalDefaultExceptionHandler 异常出现的行数:  310
14:33:01.583 http-nio-8093-exec-2  o.h.engine.jdbc.spi.SqlExceptionHelper Field 'id' doesn't have a default value
14:33:01.587 http-nio-8093-exec-2  config.GlobalDefaultExceptionHandler 异常的类型:  org.springframework.orm.jpa.JpaSystemException: could not execute statement; nested exception is org.hibernate.exception.GenericJDBCException: could not execute statement.
14:33:01.588 http-nio-8093-exec-2  config.GlobalDefaultExceptionHandler 异常出现类的名字:  org.springframework.orm.jpa.vendor.HibernateJpaDialect.
14:33:01.588 http-nio-8093-exec-2  config.GlobalDefaultExceptionHandler 异常方法的名字:  convertHibernateAccessException.
14:33:01.588 http-nio-8093-exec-2  config.GlobalDefaultExceptionHandler 异常出现的行数:  310
14:33:01.744 http-nio-8093-exec-4  o.h.engine.jdbc.spi.SqlExceptionHelper Field 'id' doesn't have a default value
14:33:01.748 http-nio-8093-exec-4  config.GlobalDefaultExceptionHandler 异常的类型:  org.springframework.orm.jpa.JpaSystemException: could not execute statement; nested exception is org.hibernate.exception.GenericJDBCException: could not execute statement.
14:33:01.749 http-nio-8093-exec-4  config.GlobalDefaultExceptionHandler 异常出现类的名字:  org.springframework.orm.jpa.vendor.HibernateJpaDialect.
14:33:01.749 http-nio-8093-exec-4  config.GlobalDefaultExceptionHandler 异常方法的名字:  convertHibernateAccessException.
14:33:01.749 http-nio-8093-exec-4  config.GlobalDefaultExceptionHandler 异常出现的行数:  310
14:34:50.985 http-nio-8093-exec-8  o.h.engine.jdbc.spi.SqlExceptionHelper Field 'id' doesn't have a default value
14:34:50.998 http-nio-8093-exec-8  config.GlobalDefaultExceptionHandler 异常的类型:  org.springframework.orm.jpa.JpaSystemException: could not execute statement; nested exception is org.hibernate.exception.GenericJDBCException: could not execute statement.
14:34:50.998 http-nio-8093-exec-8  config.GlobalDefaultExceptionHandler 异常出现类的名字:  org.springframework.orm.jpa.vendor.HibernateJpaDialect.
14:34:50.998 http-nio-8093-exec-8  config.GlobalDefaultExceptionHandler 异常方法的名字:  convertHibernateAccessException.
14:34:50.999 http-nio-8093-exec-8  config.GlobalDefaultExceptionHandler 异常出现的行数:  310
14:40:45.481 http-nio-8093-exec-3  o.h.engine.jdbc.spi.SqlExceptionHelper Field 'id' doesn't have a default value
14:40:45.493 http-nio-8093-exec-3  config.GlobalDefaultExceptionHandler 异常的类型:  org.springframework.orm.jpa.JpaSystemException: could not execute statement; nested exception is org.hibernate.exception.GenericJDBCException: could not execute statement.
14:40:45.494 http-nio-8093-exec-3  config.GlobalDefaultExceptionHandler 异常出现类的名字:  org.springframework.orm.jpa.vendor.HibernateJpaDialect.
14:40:45.494 http-nio-8093-exec-3  config.GlobalDefaultExceptionHandler 异常方法的名字:  convertHibernateAccessException.
14:40:45.494 http-nio-8093-exec-3  config.GlobalDefaultExceptionHandler 异常出现的行数:  310
14:40:49.217 http-nio-8093-exec-4  o.h.engine.jdbc.spi.SqlExceptionHelper Field 'id' doesn't have a default value
14:40:49.221 http-nio-8093-exec-4  config.GlobalDefaultExceptionHandler 异常的类型:  org.springframework.orm.jpa.JpaSystemException: could not execute statement; nested exception is org.hibernate.exception.GenericJDBCException: could not execute statement.
14:40:49.222 http-nio-8093-exec-4  config.GlobalDefaultExceptionHandler 异常出现类的名字:  org.springframework.orm.jpa.vendor.HibernateJpaDialect.
14:40:49.222 http-nio-8093-exec-4  config.GlobalDefaultExceptionHandler 异常方法的名字:  convertHibernateAccessException.
14:40:49.223 http-nio-8093-exec-4  config.GlobalDefaultExceptionHandler 异常出现的行数:  310
14:40:49.766 http-nio-8093-exec-6  o.h.engine.jdbc.spi.SqlExceptionHelper Field 'id' doesn't have a default value
14:40:49.773 http-nio-8093-exec-6  config.GlobalDefaultExceptionHandler 异常的类型:  org.springframework.orm.jpa.JpaSystemException: could not execute statement; nested exception is org.hibernate.exception.GenericJDBCException: could not execute statement.
14:40:49.773 http-nio-8093-exec-6  config.GlobalDefaultExceptionHandler 异常出现类的名字:  org.springframework.orm.jpa.vendor.HibernateJpaDialect.
14:40:49.774 http-nio-8093-exec-6  config.GlobalDefaultExceptionHandler 异常方法的名字:  convertHibernateAccessException.
14:40:49.774 http-nio-8093-exec-6  config.GlobalDefaultExceptionHandler 异常出现的行数:  310
14:40:50.358 http-nio-8093-exec-10  o.h.engine.jdbc.spi.SqlExceptionHelper Field 'id' doesn't have a default value
14:40:50.363 http-nio-8093-exec-10  config.GlobalDefaultExceptionHandler 异常的类型:  org.springframework.orm.jpa.JpaSystemException: could not execute statement; nested exception is org.hibernate.exception.GenericJDBCException: could not execute statement.
14:40:50.364 http-nio-8093-exec-10  config.GlobalDefaultExceptionHandler 异常出现类的名字:  org.springframework.orm.jpa.vendor.HibernateJpaDialect.
14:40:50.364 http-nio-8093-exec-10  config.GlobalDefaultExceptionHandler 异常方法的名字:  convertHibernateAccessException.
14:40:50.364 http-nio-8093-exec-10  config.GlobalDefaultExceptionHandler 异常出现的行数:  310
14:41:36.580 http-nio-8093-exec-2  config.GlobalDefaultExceptionHandler 异常的类型:  org.springframework.data.redis.RedisConnectionFailureException: Cannot get Jedis connection; nested exception is redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool.
14:41:36.581 http-nio-8093-exec-2  config.GlobalDefaultExceptionHandler 异常出现类的名字:  org.springframework.data.redis.connection.jedis.JedisConnectionFactory.
14:41:36.582 http-nio-8093-exec-2  config.GlobalDefaultExceptionHandler 异常方法的名字:  fetchJedisConnector.
14:41:36.582 http-nio-8093-exec-2  config.GlobalDefaultExceptionHandler 异常出现的行数:  140
